app/Http/Controllers/CompetitionsController.php

```diff
@@ -115,4 +116,18 @@ public function removeUser(Competition $competition, User $user)
 
         return redirect()->back()->with('status', 'User was removed from competition ' . $competition->id);
     }
+
+    /**
+     * Detach a user from a competition.
```

Comment:
This doesn't actually detach a user from a competition though right? Docblock copied over from above item?